### PR TITLE
Add guidance on writing good commit messages

### PR DESCRIPTION
**Pull request by 36degrees**
https://trello.com/c/X1R0jfrj/209-find-guidance-on-writing-good-commit-messages-we-can-adopt-as-a-team

### CODE REVIEW

File: development/commit-messages.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- Why is the change necessary?
+- How does the commit address the issue?
+- What are the consequences of the change?
+
+You should link to any supporting information:
+
+- relevant guidance, third party documentation or blog posts that informed
+  your decision
+- Trello cards, GitHub issues or other bug reports
+
```
Comment (hannalaakso) [inline development/commit-messages.md]: Could we add: "Trello cards/bug reports might not be viewable by everyone so don't rely on them exclusively to capture the information about your changes."

File: development/commit-messages.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Commit messages
+
+Write the summary line and description of what you have done in the imperative
+mode, that is as if you were commanding someone. For example, start the line
+with "Fix", "Add" or "Change" instead of "Fixed", "Added" or "Changed".
+
+Limit your commit summary (the first line of the commit) to 50 characters. If
+you need to provide more detail (which you often will), leave a blank line and
+add a commit description.
+
```
Comment (hannalaakso) [inline development/commit-messages.md]: Could we add: Each line in your description should wrap at 72 characters.

File: development/commit-messages.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  your decision
+- Trello cards, GitHub issues or other bug reports
+
+You should also refer to the [service manual guidance on commit
+messages](https://www.gov.uk/service-manual/technology/maintaining-version-control-in-coding#writing-commit-messages).
+
+### Further reading
+- [The art of the commit – David Demaree, A List Apart (February 2016)](https://alistapart.com/article/the-art-of-the-commit)
+- [Writing good commit messages - Erlang wiki](https://github.com/erlang/otp/wiki/Writing-good-commit-messages)
+- [How To Write A Proper Git Commit Message – Stephen Amaza (March 2017)](https://medium.com/@steveamaza/how-to-write-a-proper-git-commit-message-e028865e5791)
```
Comment (hannalaakso) [inline development/commit-messages.md]: - [What is with the 50/72 rule? - Preslav Rachev, Medium (February 2015)](https://medium.com/@preslavrachev/what-s-with-the-50-72-rule-8a906f61f09c)

**Comment by 36degrees** [comment]
@hannalaakso please re-review 🙂

**Pull request merged**